specification/monitor/resource-manager/Microsoft.Insights/preview/2023-06-01-preview/privateLinkScopes_API.json

```diff
@@ -0,0 +1,1166 @@
+          "type": "string"
+        },
+        "queryAccessMode": {
+          "description": "Specifies the access mode of queries through the specified private endpoint connection in the exclusion.",
+          "$ref": "#/definitions/AccessMode"
+        },
+        "ingestionAccessMode": {
+          "description": "Specifies the access mode of ingestion through the specified private endpoint connection in the exclusion.",
+          "$ref": "#/definitions/AccessMode"
+        }
+      }
+    },
+    "AccessMode": {
+      "description": "Access mode types.",
+      "type": "string",
+      "enum": [
+        "Open",
+        "PrivateOnly"
+      ],
+      "x-ms-enum": {
+        "name": "AccessMode",
+        "modelAsString": true
+      }
+    },
+    "AzureMonitorPrivateLinkScopeListResult": {
+      "description": "Describes the list of Azure Monitor PrivateLinkScope resources.",
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Azure Monitor PrivateLinkScope definitions.",
+          "items": {
+            "$ref": "#/definitions/AzureMonitorPrivateLinkScope"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to get the next set of Azure Monitor PrivateLinkScope definitions if too many PrivateLinkScopes where returned in the result set."
+        }
+      }
+    },
+    "ScopedResourceListResult": {
+      "description": "A list of scoped resources in a private link scope.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScopedResource"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ScopedResource": {
+      "description": "A private link scoped resource",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ScopedResourceProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "System data",
+          "readOnly": true
+        }
+      }
+    },
+    "ScopedResourceProperties": {
+      "description": "Properties of a private link scoped resource.",
+      "type": "object",
+      "properties": {
+        "kind": {
```

Comment:
>kind [](http://example.com/codeflow?start=9&length=4)

[ARMBlockingComment] Why not make use of the top level property in the envelope reserved for the same purpose?

Reply:
The kind property in Scoped Resources were introduced for this PR specifically for AMPLS, are different from the top-level property in the envelope, and serves a different purpose.

Reply:
What is the different purpose and why can that purpose not be served by the top level kind property in the envelope?
When providing the answers please provide as much context as possible to avoid back and forths.